tasks/database-sqlite3.yml

```diff
@@ -52,7 +38,7 @@
 - block:
 
     - name: Create the PowerDNS SQLite databases on Debian
-      shell: "sqlite3 {{ item }} < /usr/share/doc/pdns-backend-sqlite3/schema.sqlite3.sql"
+      shell: "sqlite3 {{ item }} < {{ pdns_deb_sqlite3_schema_file }}"
```

Comment:
I don't see why we'd have a separate variable name for each distro type, this is why we have the `vars` file that set this. I would prefer to have a `pdns_sqlite3_schema_file` (etc.) that are set in the vars by defaults.

Reply:
If I get it well you propose just two (sqlite3 and mysql schemas variables) in the defaults/main.yml file that will look like the following. Correct?
<b>
pdns_mysql_schema_file: "/usr/share/doc/pdns-backend-mysql/schema.mysql.sql"
pdns_sqlite3_schema_file: "/usr/share/doc/pdns-backend-sqlite3/schema.sqlite3.sql"
</b>

Reply:
Indedd, whose values are set via an underscore var from the `vars/*` files